tests/api/nodes_test.py

```diff
@@ -969,12 +969,27 @@ def test_adding_dimensions_to_node_columns(self, client: TestClient):
         assert response.status_code == 404
         data = response.json()
         assert data["message"] == (
-            "A column with name `non_existent_column` on node "
-            "`company_revenue` does not exist."
+            "Column non_existent_column does not " "exist on node company_revenue"
```

Comment:
What's this `" "` in the middle?

Reply:
Thanks for catching this! It was the auto-formatter that realized these two strings fit on one line, lol